huxley/www/js/components/Huxley.js

```diff
@@ -49,7 +50,11 @@ var Huxley = React.createClass({
   render: function() {
     var user = CurrentUserStore.getCurrentUser();
     if (User.isAnonymous(user)) {
-      return React.cloneElement(this.props.children, { user: user });
+      return (
+        <Shaker>
```

Comment:
This will only shake the actual div containing the content. Currently, it doesn't matter in practice because the outer views don't have any other UI like a top bar.